docs/optlib.rst

```diff
@@ -569,6 +570,144 @@ Output::
 
 "X.y" is printed as a fully qualified tag when ``--extras=+q`` is given.
 
+.. NOT REVIEWED YET (--_scopesep)
+
+Customizing scope separators
+......................................................................
+Use ``--_scopesep-<LANG>=<child-kletter>,[<parent-kletter>]:<sep>``
+option for customizing.
+
+``child-kletter``
+
+	The kind letter for a tag of inner scope.
+
+	You can use `*` for specifying as wildcards that means
+	"any kinds" for a tag of inner scope.
+
+``parent-kletter``
+
+	The kind letter for a tag of outer scope.
+
+	You can use `*` for specifying as wildcards that means
+	"any kinds" for a tag of outer scope.
+
+	If you omit ``parent-kletter``, the separator used as
+	prefix for tags specified with ``child-kletter``.
+
+``sep``
+
+
+	Separator used for the combination.
+
+Specifying `*` as both ``child-kletter`` and ``parent-kletter`` sets
```

Comment:
One other thing not clear to me is: 
Does the `--_scopesep=*,*:^` also set the language wide prefix. 
In other words, once I pass that, do I also need to pass `--_scopesep=*,:^` ?

Reply:
> Does the --_scopesep=*,*:^ also set the language wide prefix?

No. `*` doens't match (unvisible) root name space.